hal/inc/wlan_hal.h

```diff
@@ -224,6 +224,21 @@ typedef enum {
     WLAN_EAP_TYPE_PSK          = 255 /* EXPERIMENTAL - type not yet allocated draft-bersani-eap-psk-09 */
 } WLanEapType;
 
+
+// Country code enum: two ISO 3166-1 alpha-2 ASCII letters encoded as hex
+typedef enum {
+    WLAN_CC_UNSET = 0x0000,
+    WLAN_CC_US = 0x5553,
+    WLAN_CC_EU = 0x4555,
+    WLAN_CC_JP = 0x4A50,
+    WLAN_CC_CA = 0x4341,
+    WLAN_CC_MX = 0x4D58,
+    WLAN_CC_GB = 0x4742,
+    WLAN_CC_AU = 0x4155,
+    WLAN_CC_KR = 0x6B72,
+    WLAN_CC_MAX = 0xFFFF
+} wlan_country_code_t;
```

Comment:
Since this is used in dynalib as arguments, add `static_assert` on `sizeof(wlan_country_code_t)` to avoid accidentally going over the current size (we use `-fshort-enums`).